### PR TITLE
Update spec urls for some FileSystemAccess APIs and mark as standard

### PR DESCRIPTION
**Pull request by lukewarlow**
#### Summary

Large portions of the `File System Access API` now live under WhatWG's [File System API](https://fs.spec.whatwg.org/)

#### Test results and supporting details
This purely updates spec links of existing data.

#### Related issues

Fixes https://github.com/mdn/browser-compat-data/issues/13149

### CODE REVIEW

**Comment by lukewarlow** [comment]
FileSystemHandle queryPermission and requestPermission don't appear to have moved across from the Wicg version of the API so have left them as is.

**Comment by lukewarlow** [comment]
Pipeline seems to not like `https://fs.spec.whatwg.org/` not sure where I need to add this URL for the pipeline to be happy, let me know and I'll make any relevant changes.

image

**Comment by lukewarlow** [comment]
I also spotted https://github.com/mdn/browser-compat-data/issues/15536 while doing this PR so filed for future investigation

**Review by Elchi3 — COMMENTED** [review]
Looks good to me but we need to wait until we merged a release of browser-specs so that the CI can pass.

**Review by Elchi3 — APPROVED** [review]

**Pull request merged**
